modules/core/src/mathfuncs_core.cpp

```diff
@@ -373,6 +373,62 @@ void sqrt64f(const double* src, double* dst, int len)
         dst[i] = std::sqrt(src[i]);
 }
 
+// Workaround for ICE in MSVS 2015 update 3 (issue #7795)
+// CV_AVX is not used here, because generated code is faster in non-AVX mode.
+// (tested with disabled IPP on i5-6300U)
+#if (defined _MSC_VER && _MSC_VER >= 1900)
```

Comment:
I managed to run it, but it says zero tests? `Running 0 tests from 0 test cases.`